### ISSUE TITLE
Improperly insert type into literate haskell

### ISSUE THREAD

**Issue by ChristopherKing42** [atom-haskell/haskell-ghc-mod#113]
If you have

```
This is a comment.

> main = putStrLn "Hello World"

This is also a comment.
```

and ask it to insert the type of `main`, it will do

```
This is a comment.

main :: IO ()
> main = putStrLn "Hello World"

This is also a comment.
```

Correct behavior is

```
This is a comment.

> main :: IO ()
> main = putStrLn "Hello World"

This is also a comment.
```

**Comment by lierdakil** [comment]
I gather that you fixed #111 then?

**Comment by ChristopherKing42** [comment]
@liordakil This outside a cabal project. I'll have to check again for within.
